decentralised-identity.md

```diff
@@ -0,0 +1,106 @@
+3. Transparency: Blockchain is transparent by design. Every participant in the network has access to the same information. This transparency ensures that there's a shared view of the data, reducing the chances of disputes or discrepancies.
+4. Decentralization: Traditional systems often rely on a central authority to manage identity data. In blockchain-based decentralized identity, there's no single point of control. Instead, the network is distributed across many computers (nodes), making it more resilient and less susceptible to hacking or unauthorized access.
+5. User Control: Blockchain empowers individuals by giving them control over their private keys. These cryptographic keys allow you to access and manage your identity on the blockchain. As the owner of your keys, you have the ultimate say in who gets to see your data and when.
+6. Enhanced Security: The use of cryptographic techniques in blockchain ensures a high level of security. Information is encrypted and tied to your digital identity, making it difficult for unauthorized parties to gain access.
+
+Blockchain provides a robust and trustworthy foundation for decentralized identity systems, offering security, transparency, and user control.
+
+### Privacy Preservation: 
+
+Decentralized identity is all about giving you control over what you share. Here's how it increases your privacy:
+
+- Selective Disclosure: Instead of handing out your entire identity like a business card, decentralized identity lets you be picky. You can choose specific details or credentials to share, keeping the rest under wraps. It's like showing your ID without revealing your home address.
+- Need-to-Know Basis: Only share what's necessary. If someone just needs to confirm your age, that's all they get. No unnecessary details get thrown into the mix, keeping your personal info on a need-to-know basis.
+- Reduced Data Collection: Decentralized identity cuts down on the data hoarding. Traditional systems often scoop up more info than they need. With decentralized identity, the focus is on minimalism, collecting only what's essential for a particular interaction.
+- User Empowerment: You're in charge. You decide who gets what, giving you the power to manage your digital identity. This approach puts a strong emphasis on user empowerment, ensuring that you're not at the mercy of entities collecting more data than necessary.
+- Mitigating Surveillance Risks: By minimizing the information shared, decentralized identity acts as a shield against unnecessary surveillance. It aligns with the cypherpunk principles of resisting unwarranted intrusion into personal lives.
+
+In a nutshell, decentralized identity lets you share just enough to get things done while keeping the rest safely tucked away. It's privacy on your terms.
+
```

Comment:
Added the following paragraph to counteract the bias: 

Can privacy be achieved?

Selective disclosure mechanisms allow users to share specific attributes without revealing their entire identity, reducing the risk of unnecessary exposure. While DID holds great potential for enhancing privacy, its effective implementation relies on careful design, robust encryption, and user education to prevent potential pitfalls or misuses of this technology.

I suppose my utopian view would be a positive take with this article that these can be achieved. 

If there are other points to add here please feel free to unresolve